fmralign/_utils.py

```diff
@@ -44,6 +46,20 @@ def _remove_empty_labels(labels):
     return inverse_vals[labels]
 
 
+def _check_labels(unique_labels, counts, threshold=1000):
```

Comment:
maybe simply use `labels` as argument and compute `unique_labels` and `counts` inside ?